src/Dock/index.js

```diff
@@ -75,19 +107,22 @@ function Dock(props) {
 
     return {
       x: dockRect.x + window.scrollX,
-      y: dockRect.y + window.scrollY,
+      y:
+        dockRect.y +
+        window.scrollY +
+        (tabLocation === 'bottom' ? dockRect.height - dock.panelTabsHeight + 2 : 0),
     };
   })();
 
-  const width = dockRect ? dockRect.width : null;
-
+  const width = dockRect ? dockRect.width - 2 : null;
```

Comment:
Why was this 2px offset needed?

Reply:
2px was subtracted as a compensation for border. Will revert that to use outline.